src/netaddress.cpp

```diff
@@ -423,14 +461,14 @@ std::vector<unsigned char> CNetAddr::GetGroup(const std::vector<bool> &asmap) co
         }
 
         uint32_t asn = Interpret(asmap, ip_bits);
-        vchRet.push_back(nClass);
+        vchRet.push_back(net_class);
         for (int i = 0; i < 4; i++) {
             vchRet.push_back((asn >> (8 * i)) & 0xFF);
         }
         return vchRet;
     }
 
-    vchRet.push_back(nClass);
+    vchRet.push_back(net_class);
```

Comment:
this whole thing (lines 450-471) was dropped in the original PR

Reply:
I don't really follow. I think the last `vchRet.push_back(net_class);` needs to be deleted but I'm not sure.

Reply:
Pls see https://github.com/bitcoin/bitcoin/pull/16702/commits/e4658aa8eaf1629dd5af8cf7b9717a8e72028251#diff-1ce9cdfdcb1a8474c626ebd0d6eb261d2a91d4c6ac1bbed8feb57a94c61b5701L474-L496

Reply:
Resolved in 624f474391cac37594326bc6400facd385309b1e

Reply:
Hmm... I think `push_back` should go away too https://github.com/bitcoin/bitcoin/pull/16702/files#diff-1ce9cdfdcb1a8474c626ebd0d6eb261d2a91d4c6ac1bbed8feb57a94c61b5701L473-L474

Reply:
Done in 9a7881116193aeffb11b3f973a68283b28755423